packages/@uppy/url/src/Url.jsx

```diff
@@ -48,13 +48,16 @@ function getFileNameFromUrl (url) {
   const { pathname } = new URL(url)
   return pathname.substring(pathname.lastIndexOf('/') + 1)
 }
+
 /**
  * Url
  *
  */
 export default class Url extends UIPlugin {
   static VERSION = packageJson.version
 
+  static requestClientId = Url.name
```

Comment:
nit
```suggestion
  static requestClientId = this.name
```

Reply:
You sure that will work? doesn't `this` imply we have an instance? `static` happens before we have an instance right?